cryptor/cryptor.go

```diff
@@ -742,6 +742,11 @@ var ErrRestoreDelegations = errors.New("cryptor: need more delegations")
 // enough delegations are present to restore the cache, the current
 // Red October key cache is replaced with the persisted one.
 func (c *Cryptor) Restore(name, password string, uses int, slot, durationString string) error {
+	// If the persistence store is already active, don't proceed.
+	if st := c.persist.Status(); st != nil && st.State == persist.Active {
+		return nil
```

Comment:
You could catch the error before passing it back, treating it as success. The point is that you're hiding information from the caller here, when you could be passing that information up from what is effectively a library routine.